index.html

```diff
@@ -593,6 +593,28 @@ <h3>MerkleProof2019</h3>
           </pre>
       </section>
 
+      <section id="X25519KeyAgreementKey2019" about="https://w3id.org/security#X25519KeyAgreementKey2019"
+      typeof="rdfs:Class">
+        <h3>X25519KeyAgreementKey2019</h3>
+        <p>
+          This class represents a verification key.
```

Comment:
It's more than just a verification key, right? We need to point out that it's a very specific type of verification key and explain that it's used for key agreement using Curve25519 -> X25519.

Also, needs to be dealt with in a separate PR, but isn't X25519KeyAgreementKey2020 (using publicKeyMultibase) missing?

Reply:
@msporny I opened https://github.com/w3c-ccg/security-vocab/issues/111

Reply:
> We need to point out that it's a very specific type of verification key and explain that it's used for key agreement using Curve25519 -> X25519.

No we don't, how you get an X25519 key is orthogonal to defining what it is.

For example, I might have gotten one by using a mnemonic or by directly generating it from randomness, in fact our did key implementation for X25519 by default does not generate it from Ed25519.... for example:

https://did.key.transmute.industries/did:key:z6LStaHNaMZzqpABWwzUumEZyqK7xT9dLBMi2tRp5FvVxLtW

In the DID Core WG we agreed to call keyAgreement a "verification relationship" built with "verification material":

https://w3c.github.io/did-core/#key-agreement
https://w3c.github.io/did-core/#verification-material

Can you propose language that you would prefer to see in stead of "This class represents a verification key."

I would assert that "This class represents a verification key."  is the least controversial thing we might say about a key that is used for key agreement, give it allows you to verify that ECDH is possible, and it makes clear you are talking about a public component.

Reply:
I will say, I appreciate the pedantry around naming... its hard... but I also think this ship has essentially sailed thanks to the did core normative defs above... particularly "verification relationships" and "verification material"... I think we now have to align with that... or risk being viewed as undermining that work.